lib/migration.pm

```diff
@@ -35,6 +35,11 @@ our @EXPORT = qw(
 sub setup_sle {
     select_console 'root-console';
 
+    if (is_ppc64le && is_sle('<=12-sp5')) {
+        record_soft_failure('bsc#1195046 - ncurses display a wrong checker board character');
```

Comment:
For me, I think we can see this as a test requirement while not a workaround, so please remove the softfail.

Reply:
Done.

Reply:
It was solved for sle 12 sp5, please keep the workaround for sle 12 sp4 only.
https://bugzilla.suse.com/show_bug.cgi?id=1205290#c3

Reply:
Done.